wiki/Registration/de.md

```diff
@@ -1,48 +1,22 @@
----
-outdated: true
----
-Registrierung
-===========
+# Registrierung
 
-Wie registriere ich mich bei osu! ?
------------------------------------
+*Warnung: Mehr als einen osu!-Account zu besitzen, ist ein Verstoß gegen die allgemeinen [Regeln](/wiki/Rules)!*
 
-Die Registierung in osu! ist so einfach, wie das eins, zwei, drei! Befolge bitte die unten genannten Schritte.
+1. Nach der [Installation](/wiki/Installation) und dem Starten von osu! fordert dich das Spiel zum Einloggen oder Registrieren auf. Zum Erstellen eines Accounts musst du nur auf `Create an Account` klicken und osu! im Hintergrund offen lassen. Nun sollte sich die [Account erstellen](/p/register)-Webseite öffnen.
```

Comment:
"fordert dich das Spiel zum Einloggen oder Registrieren auf." Ich finde "fordert das Spiel dich auf dich einzuloggen oder zu registrieren." würde besser klingen.
"Account erstellen-Webseite" sieht doof aus da der Begriff Account und erstellen nicht miteinander verbunden ist, erstellen und Webseite allerdings schon (ich ignoriere jetzt mal das die Groß-/Kleinschreibung von diesem Begriff seltsam aussieht). Wie wäre es mit "die Webseite zum erstellen eines Accountes öffnen."

Reply:
1. Why not.. seh da jetzt keinen riesigen unterschied
2. haste recht... entweder "Account-erstellen-Webseite" oder so, und das geht gar nicht. Und so was wie "Accounterstellungsseite" find ich auch zu lang. Dann eher ne Umschreibung. Ich überleg mir was

Reply:
Beim ersten gibt es auch nicht wirklich einen großen Unterschied, es klingt einfach nur ein wenig anders. Beide Sachen gehen, gucken wir einfach was die anderen dazu sagen.